recipes/skbase/meta.yaml

```diff
@@ -0,0 +1,89 @@
+{% set name = "skbase" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}-suite
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/skbase-{{ version }}.tar.gz
+  sha256: 0f1c113fe094725f22619b30c6c8093ec77027d937adf1c2246de9344f10228e
+
+build:
+  number: 0
+  # noarch: python
+
+requirements:
+  build:
+    - python
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+outputs:
+  - name: {{ name|lower }}
+    script: build_script.sh  # [unix]
+    script: build_script.bat  # [win]
```

Comment:
Any reason to use 1-line build scripts here rather than just following the example?
https://github.com/conda-forge/staged-recipes/blob/9f47f46609da2cdf0b194c9778df5ff653c4965f/recipes/example/meta.yaml#L38

It doesn't hurt - just seems like it's more maintenance overhead for no gain?

I think `--no-build-isolation` is configured by default so you may not need it here

Reply:
@dhirschfeld I updated the PR to attempt this. But I get errors when attempting to do that. Looks like interpretation of templating for {{ PYTHON }}. Any suggestions?

Reply:
It seems odd since it's the recommended syntax. Not too sure what's going on with that.

Reply:
@dhirschfeld I'm not an expert with conda builds by any means, but I couldn't figure out what leading to the errors I received.

This setup closely follows the meta.yaml used by `sktime`s [conda-forge-feedstock](https://github.com/conda-forge/sktime-feedstock/blob/main/recipe/meta.yaml).

Given that using the separate build scripts passes the CI. Does it make sense to leave that as is for the time being? (Although I would also like to know why that's happening).

Reply:
I'm curious why it doesn't work by default, but it's fine like this.

Reply:
@dhirschfeld that sounds good. Let me know if there is anything else I need to do on my end or if this will just get picked up and merged as part of the review process.

Reply:
> I think --no-build-isolation is configured by default so you may not need it here

For multiple outputs you need that due to a bug in conda that doesn't read the default setup.